app/src/main/java/com/netcetera/skopjepulse/countryCitySelector/CountryCityFilter.kt

```diff
@@ -38,6 +40,10 @@ object CountryCityFilter{
         }
       }
     }
+
+    if(resultList.size==0)
+      resultList.add(CountryItem("No results found", "",ArrayList()))
```

Comment:
Such non-ideal scenario shouldn't be modeled like this, we call this item for a reason `CountryItem` or `CityItem`, it is because it is either `Country` or `City`. 

The end `resultList` should remain empty, but in the adapter we should:
- Return `1` for `getItemCount` when the `data` is empty.
- `getItemViewType` should have implementation that returns `EMPTY_LIST_PLACEHOLDER` when the list is empty.
- `onCreateViewHolder` and `onBindViewHolder` should have appropriate implementation that handles `EMPTY_LIST_PLACEHOLDER`.

Reply:
Fixed: [Fix empty data display](https://github.com/pulse-eco/pulse-eco-android/pull/5/commits/ae1ca2a1d750a520587174f33d32fee8b4511f88)